UPGRADE.md

```diff
@@ -1,5 +1,21 @@
 # Upgrade to 3.0
 
+## BC BREAK: Removed support for PostgreSQL 9.3 and older
+
+DBAL now requires PostgeSQL 9.4 or newer, support for unmaintained versions has been dropped.
+If you are using any of the legacy versions, you have to upgrade to newer PostgreSQL version (9.6+ is recommended).
+`Doctrine\DBAL\Platforms\PostgreSqlPlatform` and `Doctrine\DBAL\Platforms\Keywords\PostgreSQLKeywords` now represent the PostgreSQL 9.4.
```

Comment:
```suggestion
`Doctrine\DBAL\Platforms\PostgreSqlPlatform` and `Doctrine\DBAL\Platforms\Keywords\PostgreSQLKeywords` now represent PostgreSQL 9.4.
```
or did you mean to write "the state of things in PostgreSQL 9.4" or similar?

Reply:
It was backported as is from https://github.com/doctrine/dbal/pull/3477.